src/js/jsx/sections/layers/LayerGroup.jsx

```diff
@@ -157,7 +157,8 @@ define(function (require, exports, module) {
          */
         _scrollIntoFirstSelectedLayer: function () {
             var firstSelectedLayer = this.props.document.layers.selected.first(),
-                layerNode = ReactDOM.findDOMNode(this.refs[firstSelectedLayer.key]);
+                layerKey = firstSelectedLayer ? firstSelectedLayer.key : null,
+                layerNode = ReactDOM.findDOMNode(this.refs[layerKey]);
```

Comment:
Hmm, I guess you tried this and it works, but it's kind of weird to do `ReactDOM.findDOMNode(this.refs[null])`. To make this a little less scary, can we just return early if `!layerKey`?